src/acl/external/kerberos_sid_group/ext_kerberos_sid_group_acl.pl.in

```diff
@@ -0,0 +1,252 @@
+#!@PERL@ -w
+
+use strict;
+use Pod::Usage;
+use Getopt::Long;
+
+=pod
+
+=head1 NAME
+
+ ext_kerberos_sid_group_acl - external ACL helper for Squid to verify AD Domain group membership using sid.
+
+
+=head1 SYNOPSIS
+
+ ext_kerberos_sid_group_acl [-d] [-h] -p Principal Name -D Domain Controller -b Base DN -G Group1:Group2 
+
+=head1 DESCRIPTION
+
+B<ext_kerberos_sid_group_acl> is an installed executable script.
+It uses B<ldapsearch> from Openldap to lookup the name of a AD group sid.
+
+This helper must be used in with the negotiate_kerberos_auth helper in a
+Microsft AD or Samba environement.
+
+It reads from the standard input the domain username and a list of group sids
+and tries to match the group SIDs to the AD group sids.
+
+=head1 OPTIONS
+
+=over 12
+
+=item B<-d>
+
+Write debug info to stderr.
+
+=item B<-h>
+
+Print the help.
+
+=item B<-p principal name>
+
+Principal name in squid keytab to use for ldap authentication to AD 
+
+=item B<-D domain controller>
+
+Domain controller to contact to lookup group SID
+
+=item B<-b base DN>
+
+Base DN for ldap search
+
+=item B<-G AD group name>
+
+AD group name to be used for SID lookup. List seperated by :
+
+=back
+
+=head1 CONFIGURATION
+
+  auth_param negotiate program /path/to/negotiate_wrapper_auth -d --ntlm /path/to/ntlm_auth --helper-protocol=squid-2.5-ntlmssp --domain example.com --kerberos /path/to/negotiate_kerberos_auth -d -s GSS_C_NO_NAME -k /path/to/squid.keytab -t none
```

Comment:
FYI: squid.conf supports lien wrapping if the line ends with as slash '\' and the next starts with whitespace. So you can prevent line wrapping issues by wrapping example lines to be shorter than 80 characters.